crates/ruff_linter/resources/test/fixtures/airflow/AIR302_names.py

```diff
@@ -74,11 +74,20 @@
 from airflow.security.permissions import RESOURCE_DATASET
 from airflow.sensors.base_sensor_operator import BaseSensorOperator
 from airflow.sensors.date_time_sensor import DateTimeSensor
-from airflow.sensors.external_task_sensor import (
+from airflow.sensors.external_task import (
     ExternalTaskMarker,
     ExternalTaskSensor,
     ExternalTaskSensorLink,
 )
+from airflow.sensors.external_task_sensor import (
```

Comment:
Should we also add a import for the following in the test and it shouldn't raise any error:
```from airflow.providers.standard.sensors.external_task import ExternalTaskMarker```

Reply:
this one will be moved to AIR303 instead. but I want to keep the existing rules as it is and then fix the test cases later